src/components/text-input-with-copy-button/TextInputWithCopyButton.js

```diff
@@ -216,6 +218,7 @@ class TextInputWithCopyButton extends React.PureComponent<Props, State> {
         return (
             <div className={wrapperClasses} {...copyEvent}>
                 <TextInput {...inputProps} onFocus={this.handleFocus} />
+                {rest.nodeForButton}
```

Comment:
could we destructure the prop at the top of the method?

```
const { className, nodeForButton, ...rest} = this.props;
```

then you can use it directly here without the dot notation